sbi/neural_nets/density_estimators/mixed_density_estimator.py

```diff
@@ -44,167 +44,185 @@ def __init__(
 
         self.discrete_net = discrete_net
         self.continuous_net = continuous_net
-        self.log_transform_x = log_transform_x
+        self.log_transform_input = log_transform_input
 
-    def forward(self, x: Tensor):
+    def forward(self, input: Tensor):
         raise NotImplementedError(
             """The forward method is not implemented for MNLE, use '.sample(...)' to
             generate samples though a forward pass."""
         )
 
     def sample(
-        self, context: Tensor, sample_shape: torch.Size, track_gradients: bool = False
+        self, sample_shape: torch.Size, condition: Tensor, track_gradients: bool = False
     ) -> Tensor:
         """Return sample from mixed data distribution.
 
         Args:
-            context: parameters for which to generate samples.
-            sample_shape number of samples to generate.
+            sample_shape: Shape of samples to generate.
+            condition: Condition of shape `(batch_dim, *event_shape_condition)`
 
         Returns:
-            Tensor: samples with shape (num_samples, num_data_dimensions)
+            Samples of shape `(*sample_shape, batch_dim, event_dim_input)`
         """
-        assert (
-            context.shape[0] == 1
-        ), "Samples can be generated for a single context only."
+        num_samples = torch.Size(sample_shape).numel()
+        batch_dim = condition.shape[0]
```

Comment:
similar to somewhere above. It's a nitpick, but the variable naming is a bit inconsistent here: `batch_dim` refers to the `batch_size`, and `condition_event_dim` refers to the number of dimensions of the condition. 
So, maybe change it to `batch_dim <- batch_size`?